Makefile

```diff
@@ -79,6 +79,10 @@ control-plane-operator:
 hypershift:
 	$(GO_BUILD_RECIPE) -o $(OUT_DIR)/hypershift .
 
+.PHONY: hypershift-product
+hypershift-product:
```

Comment:
instead of having hypershift-product and product-cli, let's use the same term all the time i.e product-cli